osf-cas/templates/configmap.yaml

```diff
@@ -319,6 +319,352 @@ httpd/conf.d/shib.conf: |-
+
+  ########################################################################################################################
+  # Long-term Authentication: Ticket Granting Cookie (TGC) and Ticket Granting Ticket (TGT)
+  # See https://apereo.github.io/cas/6.2.x/installation/Configuring-LongTerm-Authentication.html
+  ########################################################################################################################
+  # General Cookie Setting for Ticket Granting Cookie
+  #
+  cas.tgc.name=osf-cas
+  cas.tgc.domain={{ .Values.casDomain }}
+  cas.tgc.path=/
+  cas.tgc.comment=OSF CAS Ticket Granting Cookie
+  cas.tgc.maxAge=-1
+  cas.tgc.secure=true
+  cas.tgc.http-only=true
+  cas.tgc.same-site-policy=Lax
+  cas.tgc.pin-to-session=true
+  #
+  # Ticket Granting Cookie Specific Settings
+  #
+  cas.tgc.remember-me-max-age=P28D
+  cas.tgc.auto-configure-cookie-path=true
+  #
+  # Ticket Granting Ticket Persistence Settings
+  #
+  cas.ticket.tgt.remember-me.enabled=true
+  cas.ticket.tgt.remember-me.time-to-kill-in-seconds=7200
+  ########################################################################################################################
+
+  ########################################################################################################################
+  # Pac4j Delegated Authentication
+  # https://apereo.github.io/cas/6.2.x/integration/Delegate-Authentication.html
+  ########################################################################################################################
+  # General settings
+  #
+  cas.authn.pac4j.typed-id-used=true
+  cas.authn.pac4j.lazy-init=true
+  cas.authn.pac4j.replicate-sessions=false
+  #
+  # Non-institution clients
+  cas.authn.osf-postgres.non-institution-clients[0]=${cas.authn.pac4j.orcid.client-name}
+  #
+  # Institution clients
+  #
+  cas.authn.osf-postgres.institution-clients[0]=${cas.authn.pac4j.cas[0].client-name}
+  cas.authn.osf-postgres.institution-clients[1]=${cas.authn.pac4j.cas[1].client-name}
+  #
+  # Delegation Client: ORCiD
+  #
+  cas.authn.pac4j.orcid.id=${OAUTH_ORCID_CLIENT_ID:}
+  cas.authn.pac4j.orcid.secret=${OAUTH_ORCID_CLIENT_SECRET:}
+  cas.authn.pac4j.orcid.client-name=orcid
+  cas.authn.pac4j.orcid.enabled=true
+  cas.authn.pac4j.orcid.callback-url-type=QUERY_PARAMETER
+  #
+  # Delegation Client: CAS
+  #
+  cas.authn.pac4j.cas[0].login-url=https://accounts.staging.osf.io/login
+  cas.authn.pac4j.cas[0].client-name=stage1cas
+  cas.authn.pac4j.cas[0].protocol=SAML
+  cas.authn.pac4j.cas[0].callback-url-type=QUERY_PARAMETER
+  #
+  cas.authn.pac4j.cas[1].login-url=https://accounts.staging2.osf.io/login
+  cas.authn.pac4j.cas[1].client-name=stage2cas
+  cas.authn.pac4j.cas[1].protocol=CAS30
+  cas.authn.pac4j.cas[1].callback-url-type=QUERY_PARAMETER
+  ########################################################################################################################
+config/log4j2.xml: |-
```

Comment:
The only difference between `newCAS` and `oldCAS` is that the file is under `etc/cas/config` instead of `etc/cas`. This file is a copy of [`log4j2.xml`](https://github.com/CenterForOpenScience/osf-cas/blob/feature/basic-login/etc/cas/config/log4j2.xml).